keycloak-event-emitter/src/main/java/io/cloudtrust/keycloak/kafkaeventemitter/KafkaEventEmitterState.java

```diff
@@ -0,0 +1,37 @@
+package io.cloudtrust.keycloak.kafkaeventemitter;
+
+public class KafkaEventEmitterState {
```

Comment:
What is this awful class? I strongly dislike this state expressed as a String and checking string values...
An enum would be better...